content/articles/2020_10_5_Hacktoberfest_Or_Spamtoberfest?.md

```diff
@@ -0,0 +1,56 @@
+Title: Hacktoberfest or Spamtoberfest?
+Date: 10-5-2020
+Tags: opinion, article
+Slug: hacktoberfest-or-spamtoberfest
+Authors: Jason Gutierrez
+Summary: An opinion on contributing to open source projects during Hacktoberfest
+
+# To Hacktober or Not to Hacktober?
+
+So in light of some recent events, mostly from a Youtuber with a large following,
+there was a "wave" of "spammy" PRs (Pull Requests) where "contributors" would
+add some text without out much context and that would be considered a valid PR.
+
+I sympathize with the Youtuber and I believe they had good intentions but I believe
+this lines up with the overall mentality of Imposter Syndrome which many of us, including
+myself, share. I think the concept of contributing to an open sourced project is
+noble and some projects are only alive because of a community that enjoys the service
+or project but I think there is some underlying barrier that can scare off would be
+contributors.
+
+These contributors instead only "contribute" by correcting `README`s. Though in
+some cases this is necessary, from what I have read and experienced, this is not the case.
```

Comment:
Though, in some cases, this is necessary, from what I have read and experienced, that is not the case.